src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -0,0 +1,125 @@
+    const {translate} = useLocalize();
+    const [selectedTaxes, setSelectedTaxes] = useState<string[]>([]);
+
+    const taxesList = useMemo<PolicyForList[]>(
+        () =>
+            Object.entries(policy?.taxRates?.taxes ?? {}).map(([key, value]) => ({
+                // TODO: Clean up: check if all properties are needed
+                value: value.name,
+                text: value.name,
+                keyForList: key,
+                isSelected: !!selectedTaxes.includes(key),
+                rightElement: (
+                    // TODO: Extract this into a separate component together with WorkspaceCategoriesPage
+                    <View style={styles.flexRow}>
+                        <Text style={[styles.disabledText, styles.alignSelfCenter]}>{value.isDisabled ? translate('workspace.common.disabled') : translate('workspace.common.enabled')}</Text>
+                        <View style={[styles.p1, styles.pl2]}>
+                            <Icon
+                                src={Expensicons.ArrowRight}
+                                fill={theme.icon}
+                            />
+                        </View>
+                    </View>
+                ),
+            })),
+        [policy?.taxRates?.taxes, selectedTaxes, styles, theme.icon, translate],
+    );
+
+    const toggleTax = (tax: PolicyForList) => {
+        setSelectedTaxes((prev) => {
+            if (prev.includes(tax.keyForList)) {
+                return prev.filter((item) => item !== tax.keyForList);
+            }
+            return [...prev, tax.keyForList];
+        });
+    };
+
+    const toggleAllTaxes = () => {
+        const isAllSelected = selectedTaxes.length === taxesList.length;
+        if (isAllSelected) {
+            setSelectedTaxes([]);
+        } else {
+            setSelectedTaxes(taxesList.map((item) => item.keyForList));
+        }
+    };
+
+    const getCustomListHeader = () => (
+        <View style={[styles.flex1, styles.flexRow, styles.justifyContentBetween, styles.pl3, styles.pr9]}>
+            <Text style={styles.searchInputStyle}>{translate('common.name')}</Text>
+            <Text style={[styles.searchInputStyle, styles.textAlignCenter]}>{translate('statusPage.status')}</Text>
+        </View>
+    );
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policy?.id ?? ''}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policy?.id ?? ''}>
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    style={[styles.defaultModalContainer]}
+                    testID={WorkspaceTaxesPage.displayName}
+                    shouldShowOfflineIndicatorInWideScreen
+                >
+                    <HeaderWithBackButton
+                        icon={Illustrations.Coins}
+                        title={translate('workspace.common.taxes')}
+                        shouldShowBackButton={isSmallScreenWidth}
+                    />
```

Comment:
I'm not sure, but almost every page in Workspaces uses the `WorkspacePageWithSections` wrapper. I see that `WorkspaceTagsPage` also doesn't use it. I will check it.

Reply:
I've asked the authors of Tags about it [here](https://github.com/Expensify/App/pull/37621/files#r1515780864).

Reply:
ok

Reply:
will be changed [here](https://github.com/Expensify/App/issues/37898)